dev_locale/fake_locale

```diff
@@ -12,225 +12,58 @@
 
 import polib
 import sys
+import os
 import time
-import re
 
-# Get command line arguments
-tags = False
-if '-tags' in sys.argv:
-    del sys.argv[sys.argv.index('-tags')]
-    tags = True
+if __name__ == '__main__' and __package__ is None:
+    os.sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
-if len(sys.argv) < 3:
-    print "Usage: fake-locale messages.pot en_QQ.po"
+from dev_locale.translate import trans_str  # noqa
 
-inFile = sys.argv[1]
-outFile = sys.argv[2]
 
-# load input template
-po = polib.pofile(inFile)
-new = polib.POFile()
+def main():
+    global tags
```

Comment:
Yep good catch, missed this while refactoring. `tags` is now passed into `trans_str` as a param.